samples/portable-sdks/python/async-http-api/README.md

```diff
@@ -0,0 +1,188 @@
+# Async HTTP API Pattern
+
+## Description of the Sample
+
+This sample demonstrates the async HTTP API pattern with the Azure Durable Task Scheduler using the Python SDK. This pattern is used to manage long-running operations through HTTP endpoints, allowing clients to start an operation and check its status asynchronously without keeping an HTTP connection open.
```

Comment:
Can we somehow make it clearer that this is just an integration scenario where someone exposes the Durable Task client as a HTTP Rest API?

This sample isn't inherently about using DurableTask.